lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -934,7 +934,7 @@ export class UserAgentApplication {
       let authErr: AuthError;
       let cacheResultResponse;
 
-      if (!userContainedClaims) {
+      if (!userContainedClaims && !request.forceRefresh) {
```

Comment:
You may want to add a log message for the forceRefresh case, as we have for the userContainedClaims on line 959.

Reply:
adding